docs/myscript.js

```diff
@@ -52,12 +52,9 @@ $(window).resize(function () {
 	}
 });
 
-var ispopupopen = false;
-
 mymap.on('popupopen', function (e) {
-	// curr_value=null;
-	ispopupopen = true;
 	sidebar.close();
+	curr_marker = e.target._popup._source;
```

Comment:
What would happen when someone opens a popup during the timelapse?

Reply:
It will stay visible until another popup is opened, which is an expected behavior I think